scripts/combine_all_cr.sh

```diff
@@ -0,0 +1,16 @@
+directories=$(ls -l ./charts/ | grep ^d | awk '{print $9}')
+for directory in $directories
+do
+	echo -e "---\n" > ./charts/$directory/experiments.yaml
+	subDirectories=$(ls -l ./charts/$directory | grep ^d | awk '{print $9}')
+	for subDirectory in $subDirectories
+	do 
+	cat ./charts/$directory/$subDirectory/experiment.yaml >> ./charts/$directory/experiments.yaml
```

Comment:
Can we add a failure condition here? There maybe subdirectories without the experiment yamls also in future